processor/workers.go

```diff
@@ -545,12 +562,19 @@ func CountStats(fileJob *FileJob) {
 
 	if MinifiedGenerated {
 		avgLineByteCount := len(fileJob.Content) / int(fileJob.Lines)
-		minifiedGeneratedCheck(avgLineByteCount, fileJob)
+		minifiedGeneratedCheck(avgLineByteCount, fileJob, generatedComment)
 	}
 }
 
-func minifiedGeneratedCheck(avgLineByteCount int, fileJob *FileJob) {
-	if avgLineByteCount >= MinifiedGeneratedLineByteLength {
+func minifiedGeneratedCheck(avgLineByteCount int, fileJob *FileJob, generatedComment bool) {
+	if generatedComment {
+		fileJob.Minified = true
+		fileJob.Language = fileJob.Language + " (gen)"
+
+		if Verbose {
+			printWarn(fmt.Sprintf("%s identified as generated with heading comment", fileJob.Filename))
+		}
+	} else if avgLineByteCount >= MinifiedGeneratedLineByteLength {
```

Comment:
We saying if its generated its no minified here. 

I think that might be correct. But want to check with @dbaggerman for thoughts